### PR TITLE
Add fluent-bit dependency and config values

### PR DESCRIPTION
**Pull request by jknipper**
Adds fluent-bit chart dependency and basic configuration values to kubernikus-system.

### CODE REVIEW

File: charts/kubernikus-system/requirements.yaml [hunk 1]
```diff
@@ -14,3 +14,6 @@ dependencies:
   - name: prometheus-node-exporter
     repository: https://kubernetes-charts.storage.googleapis.com/
     version: 0.3.0
+  - name: fluent-bit
```
Comment (databus23) [inline charts/kubernikus-system/requirements.yaml]: I would like it better if we vendor forked charts in the `charts/` subdirectory. Its more approachable to further changes and also much easier to take a look what the cart actually contains.

**Comment by jknipper** [comment]
In upstream config template is broken. The following PRs exists regarding this topic:
https://github.com/helm/charts/pull/8166
https://github.com/helm/charts/pull/7878
https://github.com/helm/charts/pull/7694
https://github.com/helm/charts/pull/7413
For now only kubernetes container logs are working. It seems we have systemd journal already in place in /var/log/journal and kernel log is in there as well. We need to adapt input config to log this to Elasticsearch as well. I'm going to move the chart here and change config accordingly.

**Comment by jknipper** [comment]
I added the chart to the repo with fixed config template and systemd logging capabilities.

**Comment by jknipper** [comment]
@databus23 I added official fluent-bit chart dependency, configmap and values for configuration.

**Comment by jknipper** [comment]
> @jknipper before we can merge this we need to add the required values to cc/secrets. Can you create a PR for that?

It's already there.

**Pull request merged**